modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenResponse.java

```diff
@@ -272,9 +207,15 @@ public void setMultipleOf(Number multipleOf) {
 
     @Override
     public String toString() {
-        final StringBuilder sb = new StringBuilder("CodegenResponse{");
+        final StringBuffer sb = new StringBuffer("CodegenResponse{");
```

Comment:
We probably don't need the threading synchronization overhead of `StringBuffer` in a toString method.